extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/handler/TradfriGatewayHandler.java

```diff
@@ -169,12 +175,24 @@ public void onUpdate(JsonElement data) {
             }
         }
     }
+    
+    private void updateGatewayInfoSynchronious() {
+        // we are reusing our coap client and merely temporarily set a gateway info to call
+        deviceClient.setURI(this.gatewayInfoURI);
+        String data = deviceClient.get().getResponseText();
+        JsonObject json = new JsonParser().parse(data).getAsJsonObject();
+            
+        String firmwareVersion = json.get(VERSION).getAsString();
+        getThing().setProperty("firmwareVersion", firmwareVersion);
```

Comment:
You could use final `Thing.PROPERTY_FIRMWARE_VERSION` instead of String `"firmwareVersion"`.